sway-core/tests/sway_to_ir/break.sw

```diff
@@ -0,0 +1,13 @@
+script;
+
+fn main() {
```

Comment:
Should this return a value or is exiting at all acceptable? I'm wondering about this potentially being stuck in an infinite loop and however the test is run it might not detect it except for when a dev notices that this has been running for forever.

That being said, should this have counters to exit intentionally with a failing bool if the limit is reached? This could be another indicator that there is a bug.

Similar comments for continue.sw

Reply:
Test use a metered FuelVM, so it will error with out of gas effectively instantly in real time.

Reply:
This is simply a unit test that doesn't actually run. It's just used to verify the IR generated, so the simpler I can make it the better, hence the weird structure of loops here.